views/js/controller/creator/views/section.js

```diff
@@ -342,6 +347,75 @@ function($, _, uri, __, actions, itemRefView, rubricBlockView, templates, qtiTes
             }
 
         }
+
+        /**
+         * Set up the Blueprint property
+         * @private
+         * @param {jQueryElement} $view - the $view object containing the $select
+         */
+        function blueprintProperty($view){
+            var $select = $('[name=section-blueprint]', $view);
+            $select.select2({
+                ajax:{
+                    url: data.routes.blueprintsById,
+                    dataType: 'json',
+                    delay: 350,
+                    method: 'POST',
+                    data: function (params) {
+                        return {
+                            identifier: params // search term
+                        };
+                    },
+                    results: function (data) {
+                        return data;
+                    }
+                },
+                minimumInputLength: 3,
+                width: '100%',
+                multiple : false,
+                formatNoMatches : function(){
+                    return __('Enter a blueprint');
+                },
+                maximumInputLength : 32
+            }).on('change', function(e){
+                setBlueprint(e.val);
+            });
+
+            initBlueprint();
+            $view.on('propopen.propview', function(){
+                initBlueprint();
+            });
+
+            /**
+             * Start the blueprint editing
+             * @private
+             */
+            function initBlueprint(){
+
+                if(model.blueprint === undefined){
+                    var promise = sectionBlueprint.getBlueprint(data.routes.blueprintByTestSection, model);
```

Comment:
2 smalls issues : 
 - `var`declaration should be done at the top of the scope to reflect hoisting
 - don't call it a promise, it's a defered.

But you can just chain the call
```js
sectionBlueprint
  .getBlueprint(data.routes.blueprintByTestSection, model)
  .success(...
```